scripts/nginx.sh

```diff
@@ -12,6 +12,8 @@ server {
 
         listen 80 default_server;
 
+        client_max_body_size 1G;
```

Comment:
only question is whether nginx will be happy with 1G (docs only mention m as a unit, could be g) worth testing with nginx -t to test config